fgrpc/grpcrunner.go

```diff
@@ -99,9 +99,13 @@ func (grpcstate *GRPCRunnerResults) Run(t int) {
 	if err != nil {
 		log.Warnf("Error making grpc call: %v", err)
 		grpcstate.RetCodes[Error]++
-	} else {
-		grpcstate.RetCodes[status.String()]++
+		return false, err.Error()
```

Comment:
If it's not an error, we return it as false? Shouldn't this be if it's an error?

Reply:
yea, either my diff was odd or I was not awake properly but I swear I thought that `else` was still there. :)